bindings/nodejs/types/bridge/accountManager.ts

```diff
@@ -6,7 +6,7 @@ import type {
 import type { GenerateAddressOptions } from '../address';
 import type { WalletEvent } from '../event';
 import type { Auth, ClientOptions, Node } from '../network';
-import type { EventId } from '../participation';
+import type { EventId, ParticipationEventType } from '../participation';
```

Comment:
It would be worth thinking about how best to achieve consistency with our naming here (see #1686).